atc/db/pipeline_pauser.go

```diff
@@ -48,8 +48,8 @@ func (p *pipelinePauser) PausePipelines(ctx context.Context, daysSinceLastBuild
 						)`,
 			strconv.Itoa(daysSinceLastBuild)+" day"),
 		// Covers edge case where pipelines that were just set could be paused automatically
-		// Only pauses the pipeline if it was last updated more than 24hrs ago
-		sq.Expr(`p.last_updated < CURRENT_DATE - '1 day'::INTERVAL`),
+		// Only pauses the pipeline if it was last updated more than $daysSinceCreatedWithoutBuild days ago
+		sq.Expr(`p.last_updated < CURRENT_DATE - ?::INTERVAL`, strconv.Itoa(daysSinceCreatedWithoutBuild)+" day"),
```

Comment:
Auto pauser works in a way that when a pipeline has no build for N days, then pause it. So that, if a pipeline is newly created, why only give it 1 day to create the first build? Should we also give N days to new pipeline to create the first build? @xtremerui WDYT?

Reply:
Seems reasonable to me to make them the same. I don't have the context of current value set.